image-sync/oc-mirror/Makefile

```diff
@@ -0,0 +1,14 @@
+SHELL = /bin/bash
+COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]])
+ARO_HCP_BASE_IMAGE ?= devarohcp.azurecr.io
```

Comment:
Have a look how other makefiles import dev-infrastructure/configurations/dev.mk. There is a var in there for the ACR name